299 add inference script (#338)
Project-MONAI/tutorials: kaggle/RANZCR/4th_place_solution/README.md

# Preparations

Recommend to use MONAI docker:

```
docker pull projectmonai/monai:latest
```

> Otherwise, e1xcept monai required libraries, please ensure that `opencv-python` and `scipy` are installed. The command could be:

Reviewer: ```suggestion
Otherwise, except for monai required libraries, please ensure that `opencv-python` and `scipy` are installed. The command could be:
```

python train.py -c cfg_seg_philipp_16_ch_1024_nons_full -s 701922
python train.py -c cfg_seg_philipp_16_ch_1024_nons_full -s 7259
```

# Inference

> To do inference locally, the command is like follow:

Reviewer: ```suggestion
To do inference locally, the command could be:
```
